JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -116,5 +131,30 @@ public class MessageDispatcher
 
             await _queue.Client.DeleteMessageAsync(deleteRequest);
         }
+        
+        private async Task UpdateMessageVisibilityTimeout(SQSMessage message, string receiptHandle, Message typedMessage, Exception lastException)
+        {
+            if (message.Attributes.TryGetValue(MessageSystemAttributeName.ApproximateReceiveCount, out string rawApproxReceiveCount) && int.TryParse(rawApproxReceiveCount, out int approxReceiveCount))
```

Comment:
This is a big ugly condition. can we extract method, e.g.  `int? ReadApproximateReceiveCount` ?